project/frontend/src/components/TrendingSearches/TrendingSearches.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import classes from "./TrendingSearches.module.css";
+import TrendingSearch from "./TrendingSearch/TrendingSearch";
+
+const TrendingSearches = (props) => {
+  const trends = props.searches.map((search, index) => {
+    while (index < 8) {
```

Comment:
what is the purpose of 8, make it a constant variable

Reply:
Good idea.

Reply:
Done.